### ISSUE TITLE
First Style Guide

### ISSUE THREAD

**Issue by mrmetaverse** [omigroup/website#29]
Consistent styles and branding is key to effective brand communication. OMI could benefit from a standard set of fonts, colors, styles, etc.

**Comment by mrmetaverse** [comment]
1. We have a standard logo. This was implemented (and fixed) by @jsejms when we first got started. The logo can be found in the components tab of [this figma file](https://www.figma.com/file/7IkelFTsQPpMwzrWGzqFBA/Omigroup-Website?node-id=19%3A8)

2. We have _some_ colors identified, but none officially selected, named, or shaded with various degrees of "lightness" and "darkness" 

3. We do not have a standard white, but I recommend #FCFCFC  / We do not have a standard black, but I recommend #030303 (or something slightly shifted) 

4. We do not have fonts selected 

5. We do not have a relative design system we are expanding on. However, if you find one like Material, or [One of these](https://uxplanet.org/top-12-design-systems-to-steal-from-and-up-your-design-game-520235d7de76) by all means feel free to take some of our guess work out. 

6. Do we have standard buttons? 

7. Iconography 

8. I know some folks like the hub-and-spoke design of this graphic highlighting our core values, but I don't #10 

Will add more as they come up.

**Comment by mrmetaverse** [comment]
@elirudite feel free to pull in my friend @jsejms or myself, if you want any help or want to iterate on this design system together.

**Comment by mrmetaverse** [comment]
RE: @matthewbcool  comments about designing responsive components. https://github.com/omigroup/website/issues/23#issuecomment-919588170

**Comment by mrmetaverse** [comment]
Shadows response: https://github.com/omigroup/website/issues/23#issuecomment-919657381

**Comment by mrmetaverse** [comment]
@Elirudite said today they are working on the buttons design at the moment

**Comment by mrmetaverse** [comment]
Regarding stories related to this epic: 

If you create another issue like "Design Button", just mention this ticket by typing `#29` and pressing enter. That should reference this Style guide ticket in your new "Design Button" ticket, and it will automatically add a reference to that new ticket in the comments of this Style Guide ticket. Like so: 

![image](https://user-images.githubusercontent.com/63426722/136078458-64533ad8-cba3-4e66-ad4e-a0c2dd7bc31c.png)

In the "Identify What needs to be Done" ticket #23 I mentioned this ticket #29

**Comment by Elirudite** [comment]
Apologies for the delay! It's just the same figma file as the other stuff we've been doing: https://www.figma.com/file/7IkelFTsQPpMwzrWGzqFBA/Omigroup-Designs?node-id=363%3A115

And I'm down for an accessibility review. But probably need to add more components which I'll try to do today after work.

**Comment by This-Is-Alex** [comment]
I have some suggestions for the buttons on the latest styleguide:

-  The dominant "Medium Slate" purple is dark to the point that the black text may have readability issues as the contrast is low. I suggest having white text for the buttons. Having a look at discord's UI for inspiration: the color they use for buttons is brighter, yet they still use white text on their buttons

- The text on a disabled button has a lot of contrast, whereas on other websites and UIs the constrast of the text is reduced compared to a clickable button. I suggest making the text color grey on disabled buttons (or reducing the opacity of the text, effectively achieving the same thing)

- In my opinion the shadows on the buttons are emphasised too much for viewing on a desktop: particularly the focused/active button. This could possibly be changed by reducing the opacity of the shadow, which also improves how the shadow will look on a non-white background. A grey shadow might appear strange if it's not tinted green (by reducing opacity) while on a green background, for example.
Another style I have seen that might be interesting, instead of a shadow, a thick darkened border along the bottom edge of a button

- Commenting on the secondary button: Having a border around a button clashes with having a shadow, where I think it should only have one or the other (unless the shadow is very subtle). In my opinion, a 1px border (or mobile equivalent) around a button is a future-proof design that will look clean for years to come, whereas shadows on buttons is starting to become out of fashion and is really hard to do right, as different button colors and sizes affect how visible the shadow is

- Underlining text, such as with the tertiary button, is another thing that modern websites are starting to do without. The hyperlinks within this discussion thread are one example of this, where the underline is shown when hovering over the text, but otherwise is only distinguishable as a hyperlink because of the different color

**Comment by Elirudite** [comment]
Here is the updated Styleguide! I tried to improve what @This-Is-Alex mentioned above wherever appropriate (the UI buttons have more contrast according to the Able plugin on Figma with black text than with white text, so I kept the colors the same). 

Please let me know what you all think, especially with the Tertiary buttons. I wanted to make it obvious that it is a clickable area, which having just a colored word does not do. You can see this better in the updated draft I shared here:

Feel free to [copy and update the styleguide on Figma](https://www.figma.com/file/7IkelFTsQPpMwzrWGzqFBA/Omigroup-Designs?node-id=363%3A115) yourselves though! Once again, I am not a visual designer, just applying what I know/learned about accessibility, gestalt principles, and UX to this design.

![Draft Styleguide 2](https://user-images.githubusercontent.com/10410478/141053659-9fc63112-fe51-4a56-8a26-57d715cd486c.png)

**Comment by mrmetaverse** [comment]
@Elirudite I would consider this to be nearly "done," and just blocked by #39 (Accessibility review)

**Comment by Elirudite** [comment]
That is correct!

Here is the most updated styleguide: ![Draft Styleguide 4](https://user-images.githubusercontent.com/10410478/155532064-e6b6bd7d-02d8-48b9-aaec-2282448f1595.png)
